src/main/java/com/iridium/iridiumskyblock/IridiumSkyblock.java

```diff
@@ -118,6 +119,9 @@ public void onLoad() {
     public void onEnable() {
         super.onEnable();
 
+        // Include PaperLib
+        PaperLib.suggestPaper(this);
```

Comment:
i don't think we wants this but wait to @PeachesMLG

Reply:
ye id rather not have this since spigot deems it as advertising

Reply:
Remove this please as spigot deems it as advertising